image_services/glo_30_hand/make_hand_service.py

```diff
@@ -0,0 +1,173 @@
+    clip_to_footprints='CLIP',
+    clip_to_boundary='CLIP',
+    color_correction='NOT_APPLY',
+    footprints_may_contain_nodata='FOOTPRINTS_MAY_CONTAIN_NODATA',
+    allowed_mensuration_capabilities='BASIC',
+    default_mensuration_capabilities='BASIC',
+    allowed_mosaic_methods='Center;NorthWest;Nadir;LockRaster;ByAttribute;Seamline;None',
+    default_mosaic_method='ByAttribute',
+    order_field='Name',
+    sorting_order='Ascending',
+    mosaic_operator='FIRST',
+    blend_width=10,
+    view_point_x=300,
+    view_point_y=300,
+    max_num_per_mosaic=50,
+    cell_size_tolerance=1.8,
+    cell_size=3,
+    metadata_level='BASIC',
+    transmission_fields='Name;MinPS;MaxPS;LowPS;HighPS;ZOrder;Dataset_ID;CenterX;CenterY;Tag;Tile;ProductName;DownloadURL;URLDisplay',
+    use_time='DISABLED',
+    max_num_of_download_items=50,
+    max_num_of_records_returned=2000,
+    processing_templates=f'{raster_function_template};None',
+    default_processing_template=raster_function_template,
+)
+
+arcpy.management.CalculateCellSizeRanges(
+    in_mosaic_dataset=mosaic_dataset,
+    do_compute_min='NO_MIN_CELL_SIZES',
+    do_compute_max='NO_MAX_CELL_SIZES',
+    max_range_factor=10,
+    cell_size_tolerance_factor=0.8,
+    update_missing_only='UPDATE_ALL',
+)
+
+with arcpy.EnvManager(cellSize=600):
+    arcpy.management.CopyRaster(
+        in_raster=mosaic_dataset,
+        out_rasterdataset=local_overview,
+    )
+
+subprocess.run(['aws', 's3', 'cp', local_overview, s3_overview.replace("/vsis3/", "s3://"), '--recursive'])
+
+arcpy.management.AddRastersToMosaicDataset(
+    in_mosaic_dataset=mosaic_dataset,
+    raster_type='Raster Dataset',
+    input_path=s3_overview,
+)
+
+selection = arcpy.management.SelectLayerByAttribute(
+    in_layer_or_view=mosaic_dataset,
+    selection_type='NEW_SELECTION',
+    where_clause=f"Name = '{overview_name}'",
+)
+arcpy.management.CalculateFields(
+    in_table=selection,
+    fields=[
+        ['MinPS', '600'],
+        ['Category', '2'],
+        ['ProductName', f'"{overview_name}"'],
+        ['Tag', f'"{overview_name}"'],
+        ['Tile', '"Zoom in further to see specific tile information"'],
+        ['URLDisplay', '"Zoom in further to access download link"'],
+    ],
+)
+
+arcpy.CreateImageSDDraft(
+    raster_or_mosaic_layer=mosaic_dataset,
+    out_sddraft=service_definition_draft,
+    service_name=dataset_name,
+    summary="Height Above Nearest Drainage (HAND) is a terrain model that normalizes topography to the relative " \
+            "heights along the drainage network and is used to describe the relative soil gravitational potentials " \
+            "or the local drainage potentials. Each pixel value represents the vertical distance to the nearest " \
+            "drainage. The HAND data provides near-worldwide land coverage at 30 meters and was produced from " \
+            "the 2021 release of the Copernicus GLO-30 Public DEM as distributed in the Registry of Open Data on " \
```

Comment:
Should we double check the release version given the issue we identified for `hyp3-docs`? Or do we really mean the `2021` release this time?

Reply:
https://registry.opendata.aws/copernicus-dem/

>Data is provided as Cloud Optimized GeoTIFFs and comes from Copernicus DEM 2021 release.